cmd/main.go

```diff
@@ -130,7 +128,12 @@ func main() {
 		klog.Infof("Profiler listening for requests at %s", profilerAddress)
 
 		go func() {
-			klog.Info(http.ListenAndServe(profilerAddress, nil))
+			server := &http.Server{
+				Addr:              profilerAddress,
+				ReadHeaderTimeout: 3 * time.Second,
+			}
```

Comment:
interesting, was this also reported as issue in linter? How are we coming up with 3 seconds for `ReadHeaderTimeout`

Reply:
yep, it was reported by `gosec` linter. I thought that 3 second timeout is sane...